assets/src/edit-story/components/helpCenter/karma/helpCenter.karma.js

```diff
@@ -0,0 +1,223 @@
+
+      waitFor(() => expect(toggleButton).toHaveTextContent('Help7'));
+    });
+  });
+
+  describe('Help Center cursor interaction', () => {
+    it('should toggle the Help Center closed and open again with cursor', async () => {
+      const { quickTips, toggleButton } = fixture.editor.helpCenter;
+      expect(quickTips).toBeDefined();
+
+      await fixture.events.click(toggleButton);
+      await fixture.events.sleep(500);
+      expect(fixture.editor.helpCenter.quickTips).toBeNull();
+
+      await fixture.events.click(toggleButton);
+      await fixture.events.sleep(500);
+      expect(fixture.editor.helpCenter.quickTips).toBeDefined();
+    });
+
+    it('should close the Help Center when the "close" button is clicked', async () => {
+      const { getByRole } = within(fixture.editor.helpCenter.quickTips);
+
+      const closeButton = getByRole('button', { name: /Close/ });
+      expect(closeButton).toBeTruthy();
+      await fixture.events.click(closeButton);
+      await fixture.events.sleep(500);
+      expect(fixture.editor.helpCenter.quickTips).toBeNull();
+    });
+
+    it('should update remaining unread tips as they are clicked through', async () => {
+      const { quickTips, toggleButton } = fixture.editor.helpCenter;
+      expect(toggleButton).toHaveTextContent('Help8');
+
+      const { getByLabelText } = within(quickTips);
+
+      const mainMenu = getByLabelText('Help Center Main Menu');
+      expect(mainMenu).toBeTruthy();
+
+      const { queryAllByRole } = within(mainMenu);
+
+      const tips = queryAllByRole('button');
+
+      await fixture.events.click(tips[0]);
+
+      // click through all tips
+      waitFor(() => {
+        expect(toggleButton).toHaveTextContent('Help7');
+      });
+
+      const nextButton = fixture.screen.getByRole('button', { name: /^Next$/ });
+      expect(nextButton).toBeTruthy();
+
+      await fixture.events.click(nextButton);
+      waitFor(() => {
+        expect(toggleButton).toHaveTextContent('Help6');
+      });
+
+      await fixture.events.click(nextButton);
+      waitFor(() => {
+        expect(toggleButton).toHaveTextContent('Help5');
+      });
+
+      await fixture.events.click(nextButton);
+      waitFor(() => {
+        expect(toggleButton).toHaveTextContent('Help4');
+      });
+
+      await fixture.events.click(nextButton);
+      waitFor(() => {
+        expect(toggleButton).toHaveTextContent('Help3');
+      });
+
+      await fixture.events.click(nextButton);
+      waitFor(() => {
+        expect(toggleButton).toHaveTextContent('Help2');
+      });
+
+      await fixture.events.click(nextButton);
+      waitFor(() => {
+        expect(nextButton.getAttribute('disabled')).toBeNull();
+        expect(toggleButton).toHaveTextContent('Help1');
+      });
+
+      await fixture.events.click(nextButton);
+      waitFor(() => {
+        expect(toggleButton).toHaveTextContent('Help');
+      });
+
+      await fixture.events.click(nextButton);
```

Comment:
We could probably put this into a while loop since it's always the same action
```js
let clickCount = 0;
while (toggleButton.textContent !== 'help0') {
    await fixture.events.click(nextButton);
    waitFor(() => { // does waitFor need an await?
        expect(toggleButton).toHaveTextContent(`Help${8 - clickCount}`)
    });
    clickCount++;
} 

```